K2Bridge/Visitors/Aggregations/AggregationDictionaryVisitor.cs

```diff
@@ -15,6 +16,10 @@ namespace K2Bridge.Visitors
     /// </content>
     internal partial class ElasticSearchDSLVisitor : IVisitor
     {
+        internal List<string> VisitedMetrics { get; } = new List<string>();
```

Comment:
I'd like to have some comments here on these two properties

Reply:
Done.